capstone/src/main/java/com/google/sps/service/DatabaseService.java

```diff
@@ -19,8 +23,20 @@ public static Room getRoom(Key key) throws EntityNotFoundException {
         return new Room(DatastoreServiceFactory.getDatastoreService().get(key));  
     }
 
-    public static Form getForm(Key key) throws EntityNotFoundException {
-        return new Form(DatastoreServiceFactory.getDatastoreService().get(key));
+    public static Lesson getLesson(Key key) throws EntityNotFoundException {
+        Entity entity = DatastoreServiceFactory.getDatastoreService().get(key);
+        switch((String) entity.getProperty(Lesson.TYPE_PROPERTY_KEY)) {
```

Comment:
Same as deserializeJson

Reply:
Temporal comments should be made in Github (like "Im going to fix this in the next PR" or this hack should be unecessary in a week" etc), and content comments should be made as an actual code comment so that they can be seen forever

Reply:
For sure.